src/class/ConnectError.js

```diff
@@ -51,7 +51,12 @@ export default class ConnectError extends Error {
    */
   constructor(message, code = ERRORS.NOT_DEFINED) {
     super(message);
-    this.stack = new Error().stack;
+    this.name = 'ConnectError';
```

Comment:
Why we need `name` in Error?

Reply:
Error should have name property for see in logs not `Error: some message`, but `ConnectError: some message`